### PR TITLE
I add a new open source porject

### PR DESCRIPTION
**Pull request by guillim**
Arachnida, aims to help people scrapping the web with an interface instead of having to code.

### CODE REVIEW

**Comment by shiftkey** [comment]
@guillim thanks for your interest!
I see that you don't have any open issues in the project for others to get involved with. We also recommend tagging them with a friendly label so that you can link to them easily (like the `help wanted` URL you had previously).

I'm going to hold off on merging this until those two things are done, so please let us know when it's ready!

**Comment by guillim** [comment]
@shiftkey Got it right :)

File: _data/projects/Arachnida.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- scrapping
+- scrapper
+- upforgrabs
+- contributor
+- opensource
+- js
+- Headlesschrome
+upforgrabs:
+  name: Help wanted
+  link: https://github.com/guillim/Arachnida
```
Comment (shiftkey) [inline _data/projects/Arachnida.yml:17]: This link should point the curated issues, e.g. `https://github.com/guillim/Arachnida/labels/help%20wanted`

Are you able to update that?

File: _data/projects/Arachnida.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- crawler
+- scrapping
+- scrapper
+- upforgrabs
+- contributor
+- opensource
+- js
+- Headlesschrome
+upforgrabs:
+  name: Help wanted
```
Comment (shiftkey) [inline _data/projects/Arachnida.yml:16]: :art: minor thing - the label you are using is `help wanted` (no capital H)

**Review by shiftkey — CHANGES_REQUESTED** [review]

File: _data/projects/Arachnida.yml (hunk 1 above)
Comment (guillim) [reply _data/projects/Arachnida.yml:17]: Shit I deleted the Forked repo already (stupid me). I am going to have to do a new Pull Request to get these changes right

File: _data/projects/Arachnida.yml (hunk 2 above)
Comment (guillim) [reply _data/projects/Arachnida.yml:16]: yes you're right

**Comment by ritwik12** [comment]
@guillim Ping

File: _data/projects/Arachnida.yml (hunk 1 above)
Comment (ritwik12) [reply _data/projects/Arachnida.yml:17]: @guillim I am making the desired changes for you then, hope that's fine

**Comment by ritwik12** [comment]
@guillim I guess you have deleted the fork, please resend PR, hence, Closing this,

**Comment by guillim** [comment]
I just submitted the new PR with the according to the required changes

**Pull request closed without merging**